src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -709,7 +704,16 @@ void ResolveContext::OnNewInterface(uint32_t interfaceId, const char * fullname,
     // resolving.
     interface.fullyQualifiedDomainName = hostnameWithDomain;
 
-    interfaces.insert(std::make_pair(interfaceId, std::move(interface)));
+    std::string domainFromHostname = GetDomainFromHostName(hostnameWithDomain);
+    if (domainFromHostname.empty())
+    {
+        ChipLogError(Discovery, "Mdns: Domain from hostname is empty");
```

Comment:
Should this include the hostname in the log?

But also: are we guaranteed that the hostname will always have a domain in it?

Reply:
Confirmed with Ted the hostname should always have a domain in the resolve callback. Added hostname in the log